.github/workflows/log4cxx-cpp11.yml

```diff
@@ -45,5 +45,5 @@ jobs:
       run: |
         cd main
         cd build
-        ctest -C Debug --output-on-failure -V
```

Comment:
The `-V` flag I find is important, because if a test fails there will generally be some output that is useful.

It looks like we should be able to set it as an input, so that we can turn it on if required: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions